data/linux-bridge/0004-bridge-marker-daemonset.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  name: bridge-marker
+  namespace: kube-system
+  labels:
+    tier: node
+    app: bridge-marker
+spec:
+  template:
+    metadata:
+      labels:
+        tier: node
+        app: bridge-marker
+    spec:
+      serviceAccountName: bridge-marker
+      hostNetwork: true
+      nodeSelector:
+        beta.kubernetes.io/arch: amd64
+      tolerations:
+      - key: node-role.kubernetes.io/master
+        operator: Exists
+        effect: NoSchedule
+      containers:
+      - name: bridge-marker
+        image: quay.io/kubevirt/bridge-marker:latest
```

Comment:
has to be configurable, look at the other manifest

Reply:
done